doc/remote_api.yaml

```diff
@@ -0,0 +1,100 @@
+openapi: 3.0.3
+info:
+  title: FaceMeshEditor remote api
+  description: Describes the API a server should supply to work with this application
+  version: 1.0.0
+components:
+  schemas:
+    FaceMeshRequest:
```

Comment:
The FaceMeshRequest embodies an image (png/jpg) sent to the server for processing. The server does not necessarily save the annotations - it can but doesn't have to.